.github/workflows/test_x86_tdx.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     runs-on: self-hosted
     strategy:
       matrix:
-        image: ['asterinas/asterinas:0.16.0-20250802-tdx', 'asterinas/osdk:0.16.0-20250802-tdx']
+        image: ['asterinas/asterinas:0.16.0-20250802', 'asterinas/osdk:0.16.0-20250802']
```

Comment:
Leave this change out of this PR because the QEMU installed in this Docker image is still incompatible with TDX.